File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        ArrayList<Integer> al = new ArrayList<>();
        al.add(nums[0]);
        int a = nums[0];
        
        for(int i = 1; i < nums.length; i++){
            if(nums[i] != a){
                al.add(nums[i]);
                a = nums[i];
            }
        }
        for(int i = 0; i < al.size(); i++){
            nums[i] = al.get(i);
        }
        return al.size();
    }
}
        
    

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        if not nums:
            return 0
        j = 1
        for i in range(1, len(nums)):
            if nums[i] != nums[i - 1]:
                nums[j] = nums[i]
                j += 1
        return j
